### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by fabi1cazenave**

### CODE REVIEW

**Review by Nuclear-Squid — CHANGES_REQUESTED** [review]
I really like what I’m seeing here, though right now the dummy articles feel kind of clunky. I think it’s fine if we can find them by typing the link by hand (like it’s currently possible to get the raw json data from the layouts by searching `ergol.org/layouts/ergol.json` ; like, who cares ?), but having the raw link to the article right above what should be (but isn’t) the natural place to put such link feels weird. I’ll see if I can cook something up with Hugo for you.

(Follow-up question : Is it time to get a proper `articles` page, now that we have two types of article feed ?)

**Comment by Nuclear-Squid** [comment]
I added a commit that adds a few changes to Hugo’s template system. Basically, it now looks for a `params.redirects_to` parameter in the front matter, and if it is present, the “lire la suite” tag will point to the aforementioned parameter and the dummy page will automatically redirect to the correct site.

The summary is still rendered the same, though we *still* need that stupid useless `more` tag at the end, due to [Hugo’s stupid summary rules](https://gohugo.io/content-management/summaries/#comparison) (the summary is only wrapped in a `p` tag if we have a `more` tag)

File: www/content/mentions/geobert.fr.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title = "L’après Bépo : Ergo‑L"
+date = 2024-05-28T09:46:42+01:00
+tags = ["communauté", "mentions"]
++++
+
+Après plus de 13 ans de Bépo, je me suis intéressé à une nouvelle disposition
+clavier qui a été créé en suivant d’autres principes d’optimisation. Sa sortie
+en 1.0 est l’occasion de vous la présenter ici, avec mon vécu 🙂.
+
+<https://geobert.fr/posts/l-apres-bepo-ergo-l/>
```
[the remark this answers is not in the document]
Comment (fabi1cazenave) [reply www/content/mentions/geobert.fr.md]: Couldn’t agree more. I was just unsure how to deal with it and delayed it to another PR.

**Comment by fabi1cazenave** [comment]
Nice. Ideally, the post title should redirect to the `params.redirect_to` URL, too.

**Comment by Nuclear-Squid** [comment]
It *does*, boot up a hugo server and go to `http://localhost:1313/mentions/geobert.fr/`

**Comment by Nuclear-Squid** [comment]
Titles now directly point to the correct site. Any more changes needed ?

**Pull request merged**